framework/src/outputs/FileOutput.C

```diff
@@ -64,7 +64,14 @@ FileOutput::FileOutput(const InputParameters & parameters)
     _file_num = 0;
 
   if (isParamValid("file_base"))
+  { // Check that we are the only process or not a subapp
```

Comment:
```suggestion
  {
    // Check that we are the only process or not a subapp
```

Reply:
Sorry, just noticed this weird formatting

Reply:
fixed